front/src/features/filter/__tests__/getSelectedTimeFromURL.ts

```diff
@@ -13,15 +13,48 @@ describe("getSelectedTimeFromURL", () => {
         );
         expect(result).toEqual("Custom interval");
     });
-    it("should return 'Last hour' if dateTimeFilter is not valid", () => {
-        const startDateTime = "";
-        const endDateTime = "";
-        const dateTimeFilter = "this is not valid";
+    it("should return 'Last hour' if dateTimeFilter is not defined", () => {
+        const startDateTime = undefined;
+        const endDateTime = undefined;
+        const dateTimeFilter = undefined;
+        const result = getSelectedTimeFromURL(
+            startDateTime,
+            endDateTime,
+            dateTimeFilter,
+        );
+        expect(result).toEqual("Last hour");
+    });
+    it("should return 'Last hour' if dateTimeFilter is Last hour", () => {
+        const startDateTime = undefined;
+        const endDateTime = undefined;
+        const dateTimeFilter = "Last hour";
         const result = getSelectedTimeFromURL(
             startDateTime,
             endDateTime,
             dateTimeFilter,
         );
         expect(result).toEqual("Last hour");
     });
+    it("should return 'Last day' if dateTimeFilter is Last day", () => {
+        const startDateTime = undefined;
+        const endDateTime = undefined;
+        const dateTimeFilter = "Last day";
+        const result = getSelectedTimeFromURL(
+            startDateTime,
+            endDateTime,
+            dateTimeFilter,
+        );
+        expect(result).toEqual("Last day");
```

Comment:
these tests are very very similar, could you put them in one using `it.each()`? let me know if you need help refactoring